CMakeLists.txt

```diff
@@ -61,7 +61,8 @@ add_library(astra_driver_lib src/astra_driver.cpp)
 ament_target_dependencies(astra_driver_lib
   "rclcpp"
   "builtin_interfaces"
-  "sensor_msgs")
+  "sensor_msgs"
+  "image_geometry")
```

Comment:
Nit: put parenthesis on next line so that it's easy to add dependencies without having to deal with the closing parenthesis